lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -202,8 +203,18 @@ function REPL( options ) {
 	// Define the current workspace:
 	setNonEnumerable( this, '_currentWorkspace', 'base' );
 
-	// Initialize an internal flag indicating whether a user is entering a multi-line command:
-	setNonEnumerable( this, '_multiline_mode', false );
+	// Initialize an internal object storing user settings:
+	setNonEnumerable( this, '_settings', {} );
+	setNonEnumerable( this._settings, 'autoCloseSymbols', true );
+
+	// Initialize an internal status object for controlling multi-line mode:
+	setNonEnumerable( this, '_multiline_mode', {} );
+	setNonEnumerable( this._multiline_mode, 'status', false );
+	setNonEnumerable( this._multiline_mode, 'manual', false );
```

Comment:
The use of the name "manual" is not intuitive to me, as it seems to distinguish between when using a modifier key or not. Perhaps rename to `mode` and make the value a string, such as 

```javascript
this._multiline_mode.mode = 'modifer';
```

and for the current behavior of entering multi-line mode for incomplete expressions

```javascript
this._multiline_mode.mode = 'incomplete_expression';
```

or something similar. What I am getting at is that it would be nice to have something more self-documenting to distinguish between the handling.